charts/prometheus-postgres-exporter/values.yaml

```diff
@@ -78,16 +83,19 @@ config:
   disableSettingsMetrics: false
   autoDiscoverDatabases: false
   excludeDatabases: []
-  queries: |-
-    airflow_chart_version:
-      query: |
-        SET search_path TO houston$default, public; select count(*) from "Deployment" where version != (select mode() within group (order by version) as version_group FROM "Deployment")
-      master: false
-      cache_seconds: 30
-      metrics:
-        - unconverged:
-            usage: "GAUGE"
-            description: "How many rows in the 'version' column of 'Deployments' are not converged to the mode version"
+  # you have to be connected to the houston db for this to work, no place for to set DB
```

Comment:
I think that when I tested it the connection string included ..../astronomer_houston at the end, and so it would hook up to the houston db. Basically the issue was accounting for any valid format of the connection string

Reply:
yeah, its very hard to make this work and hopefully not  break anyones setups they have since this is an OSS helm package...

such a variance in how connection strings are constructed, what they have in them, etc makes it difficult to feel comfortable about any way to make this work for me.